src/armscan_env/wrapper.py

```diff
@@ -216,35 +363,13 @@ def create_env(self, mode: EnvMode) -> LabelmapEnv:
             apply_volume_transformation=self.apply_volume_transformation,
         )
 
+        if self.add_reward_details:
```

Comment:
this should happen after frame-stack, not before. **Important:** changing this will invalidate your trainings

Reply:
As it works right now it is not possible to do it after framestack. The additional observation does not have the same size as the stacked observations! I think it can only be done like this.

Reply:
Let's have a call tomorrow, it really should not happen before stacking, that doesn't make much sense and it unnecessarily enlarges the obs space